src/main/java/org/swarm/generator/rest/ProjectGeneratorResource.java

```diff
@@ -48,7 +48,7 @@ public ProjectGeneratorResource() {
     @GET
     @Produces("application/zip")
     public Response generate(
-            @QueryParam("sv") @DefaultValue(WILDFLY_SWARM_VERSION) String swarmVersion,
+            @QueryParam("sv") @DefaultValue(THORNTAIL_VERSION) String swarmVersion,
```

Comment:
Since we're renaming the whole thing, better rename `swarmVersion` to `thorntailVersion` too? :)

Reply:
Sure, can do that, but this isn't user-visible, so I overcame my urge and didn't touch it :-) (Believe me, I had to fight hard with myself to not update the entire generator app to Thorntail 2.0.0.Final!)

Reply:
Done.